### PR TITLE
(PDB-486) Added link for the new terminus failover docs

### PR DESCRIPTION
**Pull request by senior**
This is needed in connection with this pr: https://github.com/puppetlabs/puppetdb/pull/1212

### CODE REVIEW

**Comment by senior** [comment]
@nfagerlund as I mentioned, we should probably also just point users from config_file_puppetdb.markdown to the new puppetdb_connection.markdown over on the PuppetDB side

File: source/_includes/puppetdb_master.html [hunk 1]
```diff
@@ -29,6 +29,7 @@ <h2 id="puppetdb">PuppetDB NEXT</h2>
   <li><strong>Configuration</strong>
     <ul>
       <li><a href="/puppetdb/{{ puppetdb_version }}/configure.html">Configuring PuppetDB</a></li>
+      <li><a href="/puppetdb/{{ puppetdb_version }}/puppetdb_connection.html">Configuring PuppetDB Connection</a></li>
```
Comment (nfagerlund) [inline source/_includes/puppetdb_master.html]: What do you think about changing the title of the page? I suggest `puppetdb.conf: Configuring a Puppet/PuppetDB Connection`

Like, it's longer and uglier, but anyone can see exactly whether they need to care about that page immediately, and if someone is looking for puppetdb.conf specifically they won't have to even think about it.

Comment (senior) [inline source/_includes/puppetdb_master.html]: Fine with me, should I change it at the top of the PuppetDB PR as well?

Comment (nfagerlund) [inline source/_includes/puppetdb_master.html]: I think so, yeah. It's good for Google juice.

**Comment by nfagerlund** [comment]
I'm going to put this change in a different PR along with edits to the puppetdb.conf page, so the puppetdb team can one-stop-shop when the main PR goes in.

**Pull request closed without merging**